webapp/views.py

```diff
@@ -487,6 +498,23 @@ def account_query():
     )
 
 
+def mirrors_query():
+    """
+    A JSON endpoint to request list of Ubuntu mirrors
+    """
+    local = flask.request.args.get("local", default=False)
+
+    if not local or local == "false" or local == "False" or local == "no":
```

Comment:
No idea it would be so painful working with bool query parameters in flask.

Anyway.. I would have wrote this condition instead:
```
if local and local.lower() != "true":
    local = False
```

But it's fine. Up to you again.

Reply:
This is not a boolean for Python, it's a string